File: app/Models/Paso.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class Paso extends Model
{
    use HasFactory;
    protected $table = 'pasos';
    protected $fillable =
    [
        'id',
        'numero_orden',
        'contenido',
        'id_receta'
    ];

    protected $hidden = [
        'created_at',
        'updated_at'
    ];
}

File: models/paso.go
package models

import "time"

type Paso struct {
	ID          uint      `json:"id" gorm:"primaryKey"`
	NumeroOrden int       `json:"numero_orden"`
	Contenido   string    `json:"contenido"`
	IDReceta    uint      `json:"id_receta"`
	CreatedAt   time.Time `json:"created_at"`
	UpdatedAt   time.Time `json:"updated_at"`
}

func (Paso) TableName() string {
	return "pasos"
}
